assets/Clusters/icons.tsx

```diff
@@ -0,0 +1,114 @@
+import { CoinbaseText } from '@/styles/fonts';
+
+export const ClusterIcon = ({ count }: { count: number }) => (
+  <svg
+    width="64"
```

Comment:
can you change these width and height from image to rem? I believe you can write it as like "3rem" and it should work.

Reply:
but if it's not possible, that's fine -- go ahead and squash and merge!